examples/source/advertising-analytics/Tracking_Core_Web_Vitals.html

```diff
@@ -0,0 +1,90 @@
+author: micajuineho
+
+--->
+
+<!--
+  ## Introduction
+
+  [Core Web Vital](https://blog.amp.dev/2020/05/06/amp-web-vitals-a-better-web-together/) metrics are available to developers for their AMP pages through `amp-analytics`. Variable substitution can be used to add these metrics in to any outgoing requests mader by `amp-analytics`.
+
+  If you would like to learn more about `amp-analytics`, you may find a summary [here](/content/amp-dev/documentation/guides-and-tutorials/optimize-measure/configure-analytics/analytics_basics.md).
+-->
+<!-- -->
+<!doctype html>
+<html ⚡ lang="en">
+<head>
+  <meta charset="utf-8">
+  <link rel="canonical" href="<% canonical %>">
+  <meta name="viewport" content="width=device-width">
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+  <title>Tracking Core Web Vitals</title>
+  <!-- ## Setup -->
+  <!--  Import the amp-analytics component in the header. -->
+  <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+</head>
+<body>
+
+  <!-- ## Using `amp-analytics` to track Core Web Vitals -->
+  <!--
+    You can add the core web vital macros (${largestContentfulPaint}, ${cumulativeLayoutShift}, ${firstInputDelay}) to any of your analytics requests. These macros will resolve asynchronously and can be attached to any trigger. These macros can only be accessed through `amp-analytics`.
+  -->
+  <div>
+    <amp-analytics>
+      <script type="application/json">
+        {
+          "requests": {
+            "event": "<% base_path %>/ping?user=[= user =]&account=ampdev&event=${eventId}",
+            "pageview": "${event}&lcp=${lcp}&fid=${fid}&cls=${cls}"
+          },
+          "triggers": {
+            "trackPageview": {
+              "on": "visible",
+              "request": "pageview",
+              "vars": {
+                "eventId": "pageview",
+                "lcp": "${largestContentfulPaint}",
+                "cls": "${cumulativeLayoutShift}",
+                "fid": "${firstInputDelay}"
+              }
+            }
+          }
+        }
+      </script>
+    </amp-analytics>
+  </div>
+
+
+  <!--
+  ## Tracking Core Web Vitals while using `googleanalytics`
```

Comment:
Rephrased for better SEO

```suggestion
  ## Tracking Core Web Vitals in AMP using Google Analytics
```